app/routes/paymentdata.js

```diff
@@ -1,13 +1,13 @@
-const { getPaymentData } = require('../services/databaseService')
+const { getPaymentData } = require('../services/fuzzySearchService')
 
 module.exports = {
   method: 'GET',
   path: '/paymentdata',
   handler: async (request, h) => {
     try {
-      const { searchString, limit, offset } = request.query
-      const records = await getPaymentData(searchString, limit, offset)
-      if (!records) return h.response('No data found').code(404)
+      const { searchString, limit, offset, searchBy } = request.query
+      const records = await getPaymentData(searchString, limit, offset, searchBy)
```

Comment:
Renamed the variable to sortBy from searchBy and pushed the code. Regarding the sortBy variable - yes it is optional. If you don't pass value to it, it will ignore the external sorting. So by default it will sort the results by score. This has been tested end to end in FE, we currently get the results sorted by score.